runtime/calamari/src/weights/pallet_tx_pause.rs

```diff
@@ -54,13 +55,13 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_tx_pause::WeightInfo for SubstrateWeight<T> {
     // Storage: TransactionPause PausedTransactions (r:1 w:1)
     fn pause_transaction() -> Weight {
-        (18_286_000 as Weight)
+        (33_718_000 as Weight)
```

Comment:
weird 2x

Reply:
doesn't seem to be due to code changes in the pallet, only reason coming to mind is the deps upgrade.

I'm really starting to think whatever that "bare metal" device is we're getting from AWS is not actually fully bare. Unless we're running some tasks on it in parallel with benchmarking accidentally which would introduce all these fluctuations

Reply:
There's also the - more likely - possibility that benchmarking just sucks on that timescale.

Our billion long private extrinsics have differences between versions on the order of 100mil weight.
100mil = 100ns . A non-realtime OS will have trouble reproducibly measuring anything on that timescale anyway, so I'm starting to lean towards discarding any benchmarking diffs below that scale out of hand

Reply:
Rerunning benchmark twice to verify
- https://github.com/Manta-Network/Manta/actions/runs/3955495077
- https://github.com/Manta-Network/Manta/actions/runs/3955501543

Reply:
Both benchmarks got cancelled. Merging anyway